docs/cmdline-opts/v4-signature.d

```diff
@@ -0,0 +1,20 @@
+Long: v4-signature
+Arg: <provider1[:provider2]>
+Help: Performe AWS V4 signature on HTTP header
```

Comment:
Spelling, and it doesn't "perform" a header. It might **add** a header? Or **provide** ?